INSTALL.md

```diff
@@ -94,42 +94,51 @@ on experience with the tools involved.
 ### Building using CMake (RECOMMENDED)
 
 Builds gRPC C and C++ with boringssl.
-- Install [Git](https://git-scm.com/).
 - Install Visual Studio 2015 or 2017 (Visual C++ compiler will be used).
+- Install [Git](https://git-scm.com/).
 - Install [CMake](https://cmake.org/download/).
-- Install [Active State Perl](https://www.activestate.com/activeperl/) (`choco install activeperl`)
-- Install [Ninja](https://ninja-build.org/) (`choco install ninja`)
-- Install [Go](https://golang.org/dl/) (`choco install golang`)
-- Install [yasm](http://yasm.tortall.net/) and add it to `PATH` (`choco install yasm`)
-- Run these commands in the repo root directory
-
-#### cmake: Using Ninja (faster build, supports boringssl's assembly optimizations).
-Please note that when using Ninja, you'll still need Visual C++ (part of Visual Studio)
-installed to be able to compile the C/C++ sources.
+- Install [Active State Perl](https://www.activestate.com/activeperl/) (`choco install activeperl`) - *required by boringssl*
+- Install [Go](https://golang.org/dl/) (`choco install golang`) - *required by boringssl*
+- Install [yasm](http://yasm.tortall.net/) and add it to `PATH` (`choco install yasm`) - *required by boringssl*
+- (Optional) Install [Ninja](https://ninja-build.org/) (`choco install ninja`)
+
+#### Clone grpc sources including submodules
+Before building, you need to clone the gRPC github repository and download submodules containing source code 
+for gRPC's dependencies (that's done by the `submodule` command).
 ```
-> powershell git clone --recursive -b ((New-Object System.Net.WebClient).DownloadString(\"https://grpc.io/release\").Trim()) https://github.com/grpc/grpc
+> @rem You can also do just "git clone -b THE_BRANCH_YOU_WANT https://github.com/grpc/grpc"
```

Comment:
I feel like omitting `--recursive` in favor of explicit submodule checkout is a regression. People forget to run the submodule init and file build failed bugs

Reply:
Ok, I might change it back. The difference is that we only need recursive with limited depth (e.g. not necessary to fetch protobuf's submodules). I actually might add --recurse-depth 2

Reply:
`--recursive` is not perceptibly slower and it does not really hurt IMO. I think a shorter command line might be better but your call.